src/mon/PGMonitor.cc

```diff
@@ -1375,8 +1375,11 @@ void PGMonitor::dump_object_stat_sum(TextTable &tbl, Formatter *f,
     tbl << stringify(si_t(sum.num_bytes));
     int64_t kb_used = SHIFT_ROUND_UP(sum.num_bytes, 10);
     float used = 0.0;
-    if (pg_map.osd_sum.kb > 0)
-      used = (float)kb_used * raw_used_rate * curr_object_copies_rate / pg_map.osd_sum.kb;
+    if (avail) {
+      used = static_cast<float>(sum.num_bytes) / (sum.num_bytes + avail);
```

Comment:
LGTM :+1:  I played with "ceph df" using vstart and the change works.   

@tchaikov Did you play with rules that limit OSDs for a pool?